src/ripple/app/misc/AmendmentTableImpl.cpp

```diff
@@ -97,51 +101,114 @@ class AmendmentTableImpl : public AmendmentTable
     amendmentList_t getDesired();    // amendments we support, do not veto, are not enabled
 };
 
-void
-AmendmentTableImpl::addInitial ()
+namespace detail
 {
-    // For each amendment this version supports, construct the AmendmentState object by calling
-    // addKnown. Set any vetoes or defaults. A pointer to the AmendmentState can be stashed
+using PreEnabledAmendmentsCollection = std::vector<AmendmentName>;
```

Comment:
Is "PreEnabledAmendmentsCollection" really preferred over "std::vector<AmendmentName>;" ??